test/auth.test.js

```diff
@@ -0,0 +1,88 @@
+import 'babel-polyfill';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import models from '../src/api/models/index';
+
+import server from '../src/index';
+
+// const should = chai.should();
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Signup', () => {
+  it('should register and give the token', (done) => {
+    chai.request(server)
+      .post('/api/auth/signup')
+      .send({
+        names: 'Burindi Alain',
+        email: 'alain@gmail.com',
+        password: 'password',
+        confirmPassword: 'password'
+      })
+      .end((err, res) => {
+        res.should.have.status(201);
+        res.should.be.an('Object');
+        res.body.should.have.property('user');
+        res.body.user.should.be.an('Object');
+        res.body.user.should.have.property('token');
+        done();
+      });
+  });
+});
+
+describe('Login', () => {
+  it('should login and give the token', (done) => {
+    chai.request(server)
+      .post('/api/auth/login')
+      .send({
+        email: 'alain@gmail.com',
+        password: 'password',
+      })
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.be.an('Object');
+        res.body.should.have.property('user');
+        res.body.user.should.be.an('Object');
+        res.body.user.should.have.property('token');
```

Comment:
Please, confirm that the right token, email etc are returned.

Reply:
@alainburindi please verify the other content of the response, not only the token.

e.g  
```
    user = res.body.user
    user.username.should.equal('Burindi Alain');
    user.email.should.equal('alain@gmail.com');
```

etc

It's important to thoroughly check that the right response is returned. 
N.b this applies to other tests, as well.